sds_scraper/sds_scraper/spiders/Ivoclar_Vivadent_AG.py

```diff
@@ -0,0 +1,30 @@
+from datetime import datetime
+
+import scrapy
+from scrapy import Selector
+from scrapy.spiders import CrawlSpider
+
+
+class IvoclarVivadent(CrawlSpider):
+    name = 'ivoclar_vivadent'
+    base_url = 'https://www.ivoclarvivadent.se/'
+
+    def start_requests(self):
+        url = 'https://www.ivoclarvivadent.se/sv/skerhetsdatablad/'
+
+        yield scrapy.Request(url=url, callback=self.parse)
+
+    def parse(self, response):
+        # 1769 files
+        for items in response.xpath(
+                '//div[@class="grid__item one-half palm--one-whole"]/div[@class="item"]/div[@class="text"]').getall():
```

Comment:
try converting this xpath into equivalent css and use that, also its too long a path optimize it.